README.md

```diff
@@ -1,8 +1,65 @@
+# Country By Country Reporting Frontend
 
-# country-by-country-reporting-frontend
+## Info
 
-This is a placeholder README.md for a new repository
+This service allows a user to upload the correct tax file which further goes for validation and once validated it finally gets uploaded.
 
-### License
+This service has a corresponding back-end service, namely country-by-country-reporting.
 
-This code is open source software licensed under the [Apache 2.0 License]("http://www.apache.org/licenses/LICENSE-2.0.html").
\ No newline at end of file
+### Dependencies
+
+| Service           | Link                                                          |
+|-------------------|---------------------------------------------------------------| 
+| Address Lookup    | https://github.com/hmrc/address-lookup                        |
+| Email             | https://github.com/hmrc/email                                 |
+| Auth              | https://github.com/hmrc/auth                                  |
+| Tax Enrolments    | https://github.com/hmrc/tax-enrolments                        |
+| origin            | https://github.com/hmrc/country-by-country-reporting-frontend |
+
+### Endpoints used
+
+| Service             | HTTP Method | Route                                   | Purpose                                                          |
+|---------------------|-------------|-----------------------------------------|------------------------------------------------------------------|
+| subscription        | POST        | /subscription/read-subscription         | Enables user to read subscription details using subscription Id  |
```

Comment:
The service name needs to be the service that is being called as part of the endpoint. So in this case our "Country-by-country-reporting" service is being called for most of these endpoints so that is the service name that needs to be in this table for each endpoint